packages/react/src/components/CommandBar/CommandBar.test.tsx

```diff
@@ -83,13 +67,9 @@ describe('CommandBar', () => {
       />,
     );
 
-    const menuItem = commandBar.find('.MenuItem button');
-
-    expect(menuItem.length).toEqual(1);
-
-    menuItem.simulate('click');
-
-    expect(document.querySelector('.SubMenuClass')).toBeTruthy();
+    const menuItem = getByRole('menuitem', { hidden: true });
```

Comment:
Another case that testing-library deems as "inaccessible" for seemingly no reason 😢

Reply:
This was bothering me enough that I started looking at the testing-library code, and I figured it out...it's the measurement/overflow logic. I don't know the details of how measurement works, but the general idea is that it initially renders stuff hidden and/or off-screen to get the size, then re-renders with appropriate size and overflow items. `getByRole` is running against that initial measurement render, where the elements are in fact hidden.

How I figured this out was a little over-complicated, and in retrospect I probably should have tried running the test and stepping through the code first. 😆 

What I actually did: I started by looking in `@testing-library/dom`'s code for the implementation of the [`role` query](https://github.com/testing-library/dom-testing-library/blob/6b99a7eca1f00ee5415ec3f007b67b6ccc9d4e94/src/queries/role.js#L173) and searching for `hidden`. That led to the [`isInaccessible` implementation](https://github.com/testing-library/dom-testing-library/blob/6b99a7eca1f00ee5415ec3f007b67b6ccc9d4e94/src/role-helpers.js#L43).  I wasn't sure exactly what would cause issues there, so I made a codepen that renders a basic Breadcrumb (since it has the same problem and is a little simpler) and runs `isInaccessible` against the list items after render.

Turns out `getComputedStyle(element).visibility` was returning `'hidden'`. This seemed weird (since I could see the breadcrumb on the page) but then I remembered that Breadcrumb and CommandBar have measurement with initial hidden rendering. So I tried running `isInaccessible` on the items again after a 500ms timeout and it returns false (*is* accessible) as expected.
https://codepen.io/ecraig12345/pen/RwxZwdN?editors=0011

Anyway...the possible solutions for the tests are:
- keep `{ hidden: true }` and add comments that it's due to the measurement logic
- use fake timers and run timers before doing `getByRole` on items

Definitely any tests that involve overflow should use fake timers, but in other cases it's up to you.